frontend/src/App.tsx

```diff
@@ -73,9 +73,17 @@ const App = () => {
                       path="/access/request"
                       component={AccessRequestPage}
                     ></PrivateRoute>
-                    <PrivateRoute path="/mapView" component={MapView} />
-                    <PrivateRoute path="/submitProperty/:id?" component={SubmitProperty} />
-                    <PrivateRoute path="/edituser" component={EditUserPage} />
+                    <PrivateRoute path="/mapView" component={MapView} role={Claims.PROPERTY_VIEW} />
```

Comment:
Only allowed to view the map if the user has the claim property-view.

Reply:
shouldn't this be claim={Claims.PROPERTY_VIEW} ?

Reply:
@Fosol Will approve after you fix this.

Reply:
Fixed